server_admin/topics/clients/saml/idp-initiated-login.adoc

```diff
@@ -6,6 +6,15 @@ In the `Settings` tab for your client, you need to specify the `IDP Initiated SS
 This is a simple string with no whitespace in it.
 After this you can reference your client at the following URL: `root/auth/realms/{realm}/protocol/saml/clients/{url-name}`
 
+The IDP initiated login prefers POST binding over REDIRECT (if using REDIRECT the SAML response is part of the URL, it is as a paremeter,
```

Comment:
The part in brackets is general recommendation, not specific to IdP-initiated login. Could you please move it to https://github.com/keycloak/keycloak-documentation/blob/master/server_admin/topics/sso-protocols/saml.adoc#post-binding?